src/metabase/notification/send.clj

```diff
@@ -179,7 +179,7 @@
                             (channel-send-retrying! id payload_type handler message)))
                         (catch Exception e
                           (log/warnf e "Error sending to channel %s" (handler->channel-name handler))))))))
-              (do-after-notification-sent notification-info notification-payload)
+              (do-after-notification-sent hydrated-notification notification-payload)
```

Comment:
`do-after-notification-sent` requires fully hydrated notification to work, especially it needs the `:payload` to be hydrated.